osf/models/mixins.py

```diff
@@ -2214,6 +2214,19 @@ def expand_registration_responses(self):
             self.file_storage_resource,
         )
 
+    def copy_registration_responses_into_schema_response(self, draft_registration):
```

Comment:
Registration doesn't actually inherit from RegistrationResponseMixin. Also, this functionality is only for Registrations, not DraftRegistrations, so no reason for it to live here.

Reply:
No Registration does actually inherit this, look at the AbstractNode model.

Reply:
🤮

Reply:
The point does still hold that the DraftRegistration will (probably) never call this.